components/raftstore/src/store/peer.rs

```diff
@@ -1112,6 +1121,7 @@ where
             long_uncommitted_threshold: cfg.long_uncommitted_base_threshold.0,
             peer_cache: RefCell::new(HashMap::default()),
             peer_heartbeats: HashMap::default(),
+            wait_data_peers: Vec::default(),
```

Comment:
Question: when is this wait_data_peers be updated? I did not see it in this PR.

Reply:
Because the current update logic is related to the witness, it is placed in this PR：https://github.com/tikv/tikv/pull/13491/files#diff-7a4718b9cddd27a4ef7e563024d5283cf14b4ba3abf26edc33b974b87680a5fbR3684